lessons/introduction-to-smartpy/04/04.mdx

```diff
@@ -6,16 +6,19 @@ editor:
   language: python
   startingCode: |
     import smartpy as sp
+
     class Plant(sp.Contract):
 
+        # Code from below
 
-    
   answer: |  
     import smartpy as sp
+
     class Plant(sp.Contract):
+
```

Comment:
remove extra space here

Reply:
These spaces would make the code much more readable which is why I added these. Will remove all new lines no worries.

Reply:
ok